src/main/webapp/app/detail-overview-list/components/text-detail.component.html

```diff
@@ -0,0 +1,5 @@
+@if (detail.data.text) {
```

Comment:
Moved from typescript inline template to own file https://github.com/ls1intum/Artemis/pull/9053#discussion_r1678834517